### ISSUE TITLE
[a11y] 11.1.1.1b Non-text content (graphics and objects)

### ISSUE THREAD

**Issue by tschiebel** [owncloud/android#4361]
audit: https://infinite.owncloud.com/s/QtwuiGHikwFjozV

https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf
https://github.com/BIK-BITV/BIK-App-Test/tree/main/Pr%C3%BCfschritte/de

### Image Viewer

- [x] The images in the image viewer are not accessible for screen readers and have no alternative text.

**Comment by jesmrec** [comment]
- Translatable text
